src/css/benchmark.css

```
@@ -0,0 +1,54 @@
+.box {
+  background-color: #fff;
```

we must use color vars

```suggestion
:root {
  --fty-box-primary: var(--ifm-color-secondary-lightest);
}

[data-theme='dark'] {
  --fty-box-primary: var(--ifm-color-secondary-darkest);
}

.box {
  background-color: var(--fty-box-primary);
```